### DISCUSSION TITLE
What is the Google search engine rating for the webpage you developed using this UI library?

### DISCUSSION THREAD

**Discussion by BadDeveloper2022** [ikun-svelte/ikun-ui#308] [Q&A] [answered]
### Description

What is the Google search engine rating for the webpage you developed using this UI library?

**Comment by baiwusanyu-c** [comment, marked as answer]
Do you mean the score on seo?

**Reply by BadDeveloper2022** [reply]
Yes. I have reviewed the demo and found that there are quite a few loaded JavaScript files. Will this affect Google's rating of the page.

**Reply by baiwusanyu-c** [reply]
In fact, I must admit that we haven't conducted relevant tests at the moment. However, if you are developing an application based on Svelte Kit and have done a good job with SSR (Server-Side Rendering), you should at least have a good SEO performance, which is not directly related to ikun-ui itself. As for the issue of excessive JavaScript file loading, this has been working well in our internal projects. This is because Svelte projects generate a substantial amount of compiled code for individual components, and if you're observing it on a documentation website, it's built with Vitepress, so it also loads Vue-related code. So, it's influenced by various factors.
